scripts/tests.sh

```diff
@@ -63,8 +63,8 @@ function download_eth()
         ETH_HASH="e527dd3e3dc17b983529dd7dcfb74a0d3a5aed4e"
         if grep -i trusty /etc/lsb-release >/dev/null 2>&1
         then
-            ETH_BINARY=eth_byzantium2
-            ETH_HASH="4dc3f208475f622be7c8e53bee720e14cd254c6f"
+            ETH_BINARY=eth_byzantium_trusty
```

Comment:
Can we put here a cpp-ethereum commit hash this was built from?

Reply:
Done (we can squash while merging)

Reply:
Also is the artful build above the same or is it different?

Reply:
It is a different build, I do not remember which one exactly. I made is together with enabling endtoend tests on circle.